docs/release-notes/release-notes-0.18.0.md

```diff
@@ -1,4 +1,5 @@
 # Release Notes
+<<<<<<< HEAD
```

Comment:
I guess you are still working on this if not there is a conflict marker.

Reply:
will remove after next review round 👍